spec.md

```diff
@@ -828,30 +864,32 @@ message ListVolumesRequest {
 }
 
 message ListVolumesResponse {
-  message Result {
-    message Entry {
-      VolumeInfo volume_info = 1;
-    }
-
-    repeated Entry entries = 1;
-
-    // This token allows you to get the next page of entries for
-    // `ListVolumes` request. If the number of entries is larger than
-    // `max_entries`, use the `next_token` as a value for the
-    // `starting_token` field in the next `ListVolumes` request. This
-    // field is OPTIONAL.
-    // An empty string is equal to an unspecified field value.
-    string next_token = 2;
+  message Entry {
+    VolumeInfo volume_info = 1;
   }
 
-  // One of the following fields MUST be specified.
-  oneof reply {
-    Result result = 1;
-    Error error = 2;
-  }
+  repeated Entry entries = 1;
+
+  // This token allows you to get the next page of entries for
+  // `ListVolumes` request. If the number of entries is larger than
+  // `max_entries`, use the `next_token` as a value for the
+  // `starting_token` field in the next `ListVolumes` request. This
+  // field is OPTIONAL.
+  // An empty string is equal to an unspecified field value.
+  string next_token = 2;
 }
 ```
 
+##### ListVolumes Errors
+
+If the plugin is unable to complete the ListVolumes call successfully, it MUST return a non-ok gRPC code in the gRPC status.
+If the conditions defined below are encountered, the plugin MUST return the specified gRPC error code.
+The CO MUST implement the specified error recovery behavior when it encounters the gRPC error code.
+
+Condition | gRPC Code | Description | Recovery Behavior
+| --- | --- | --- | --- |
+| Invalid `starting_token` | 10 ABORTED | Indicates that `starting_token` is not valid. | Caller SHOULD start the `ListVolumes` operation again with an empty `starting_token`. |
```

Comment:
there's no UNIMPLEMENTED row here, like for the other optional calls. suggest adding another row for this but as ABORTED (as suggested above).